packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -399,28 +400,24 @@ export const Subscriptions = ({
                 <Localized
                   id="manage-pocket-body"
                   elems={{
-                    a: (
-                      <a
-                        target="_blank"
-                        rel="noopener noreferrer"
+                    linkExternal: (
+                      <LinkExternal
                         href="https://getpocket.com/premium/manage"
                         data-testid="manage-pocket-link"
                       >
                         click here
-                      </a>
+                      </LinkExternal>
                     ),
                   }}
                 >
                   <p>
                     To manage it,{' '}
-                    <a
-                      target="_blank"
-                      rel="noopener noreferrer"
+                    <LinkExternal
```

Comment:
At the moment the link isn't showing up for this. (See screenshot below)

Since this is changed from `<a>` tags to `<LinkExternal>` you'll need to update the `elems` property in `<Localized>`, as well as in the `main.ftl` file.

![image](https://user-images.githubusercontent.com/10620585/166745212-bfa787c9-e716-4ab2-bc06-f9c0e958f0ba.png)